code/modules/mapping/space_management/space_reservation.dm

```diff
@@ -56,6 +68,40 @@
 		SSmapping.unused_turfs["[cordon_turf.z]"] -= cordon_turf
 		SSmapping.used_turfs[cordon_turf] = src
 
+	//swap the area with the pre-cordoning area
+	for(var/turf/pre_cordon_turf as anything in pre_cordon_turfs)
+		MakeRepel(pre_cordon_turf)
+
+///Register signals in the cordon "danger zone" to do something with whoever trespasses
+/datum/turf_reservation/proc/MakeRepel(turf/pre_cordon_turf)
+	//Okay so hear me out. If we place a special turf IN the reserved area, it will be overwritten, so we can't do that
```

Comment:
```suggestion
	SHOULD_CALL_PARENT(TRUE)
	//Okay so hear me out. If we place a special turf IN the reserved area, it will be overwritten, so we can't do that
```